api.go

```diff
@@ -47,6 +47,10 @@ type FileMatch struct {
 	// Sourcegraph.
 	RepositoryID uint32
 
+	// RepositoryPriority is a Sourcegraph extension. It is used by Sourcegraph to
+	// order results from different repositories relative to each other.
+	RepositoryPriority float64
```

Comment:
Does it need to be exported?

Reply:
As discussed in our sync meeting, we could avoid exporting this field by adding the priority, EG in form of a map, to shardedSearcher. However, I tried that variant and the code felt definitely more awkward. Adding repository priority to each FileMatch seems to be a better fit.